Modelica/Thermal/HeatTransfer/UsersGuide/Contact.mo

```diff
@@ -0,0 +1,30 @@
+within Modelica.Thermal.HeatTransfer.UsersGuide;
+class Contact "Contact"
+  extends Modelica.Icons.Contact;
+  annotation (preferredView="info",Documentation(info="<html>
+<dl>
+  <dt><strong>Main Authors:</strong></dt>
+  <dd>
+  <p>
+  <a href=\"https://www.haumer.at/\">Anton Haumer</a><br>
+  Technical Consulting &amp; Electrical Engineering<br>
+  D-93049 Regensburg, Germany<br>
+  email: <a href=\"mailto:a.haumer@haumer.at\">a.haumer@haumer.at</a>
+</p>
+  </dd>
+</dl>
+<p>
+Copyright &copy; 2001-2019, Modelica Association and contributors
+</p>
+
+<p>
+<strong>Acknowledgements:</strong><br>
+Several helpful remarks from the following persons are acknowledged:
+John Batteh, Ford Motors, Dearborn, U.S.A;
+<a href=\"https://www.haumer.at/\">Anton Haumer</a>, Technical Consulting &amp; Electrical Engineering, Germany;
+Ludwig Marvan, VA TECH ELIN EBG Elektronik GmbH, Wien, Austria;
+Hans Olsson, Dassault Syst&egrave;mes AB, Sweden;
+Hubertus Tummescheit, Lund Institute of Technology, Lund, Sweden.
```

Comment:
At least John Batteh and @hubertus65 changed their affiliation. I am not sure how to deal with it, but I recommend to change it as follows:

John Batteh, previously at Ford Motors, Dearborn, U.S.A
Hubertus Tummescheit, previously at Lund Institute of Technology, Lund, Sweden